### ISSUE TITLE
SecretManagerServiceClient refers to a value, but is being used as a type here?

### ISSUE THREAD

**Issue by JustinBeckwith** [googleapis/gapic-generator-typescript#694]
I am trying to do something like this:

```ts
import { SecretManagerServiceClient } from "@google-cloud/secret-manager";
let client: SecretManagerServiceClient;
client = new SecretManagerServiceClient();
```

I was really surprised when the `client: SecretManagerServiceClient` bit didn't work.  I get an error like this:

```
'SecretManagerServiceClient' refers to a value, but is being used as a type here. Did you mean 'typeof SecretManagerServiceClient'?ts(2749)
```

I'm not 100% sure what we're doing wrong, but I had to dig the whole way into v1 to get what I wanted:
```ts
import { SecretManagerServiceClient } from "@google-cloud/secret-manager/build/src/v1";
```

**Comment by alexander-fenster** [comment]
Will be fixed as I did in https://github.com/googleapis/nodejs-speech/pull/643.

**Comment by alexander-fenster** [comment]
Fixed in #696.
